### DISCUSSION TITLE
Tolerance option

### DISCUSSION THREAD

**Discussion by dirt66merchant** [EvilHacker/BoxJoint#15] [Ideas]
This add in is so close to perfection, if you can only add an option for tolerance for these joints (clearance value!) it would take it to the level of absolute perfection.

**Comment by KarolBeaudoin** [comment]
I did the tolerance option, see if it works for you. I only tested in imperial (inches)
I dont know how to add a python file, but replace the info in "fusion_box_join.py" by the info in the document!
[tolerances.txt](https://github.com/user-attachments/files/19538376/tolerances.txt)
